webapp/app/forms/steps/lotse/fahrkostenpauschbetrag.py

```diff
@@ -1,9 +1,11 @@
-def calculate_fahrkostenpauschbetrag(has_pflegegrad=False, disability_degree=None, has_merkzeichen_bl=False, has_merkzeichen_tbl=False,
-                                     has_merkzeichen_h=False, has_merkzeichen_ag=False, has_merkzeichen_g=False):
-    if has_pflegegrad or has_merkzeichen_bl or has_merkzeichen_tbl or has_merkzeichen_tbl or has_merkzeichen_h or has_merkzeichen_ag:
+def calculate_fahrkostenpauschbetrag(has_pflegegrad: str = None, disability_degree: int = None,
```

Comment:
Kind of funny that we have to change that function in the end as we discussed twice if we could trust the input data